driver-core/src/test/java/com/datastax/driver/core/policies/DowngradingConsistencyRetryPolicyIntegrationTest.java

```diff
@@ -40,10 +40,9 @@
         numberOfNodes = 3,
         config = {
                 // tests fail often with write or read timeouts
-                "hinted_handoff_enabled:true",
                 "phi_convict_threshold:5",
-                "read_request_timeout_in_ms:100000",
```

Comment:
Seems weird to require timeouts this large, will be interesting to see if it improves things in CI.